dora/core/server/worker/src/main/java/alluxio/worker/AlluxioWorker.java

```diff
@@ -53,6 +47,7 @@ public static void main(String[] args) {
     }
 
     CommonUtils.PROCESS_TYPE.set(CommonUtils.ProcessType.WORKER);
+    /*
```

Comment:
remove commented code?

Reply:
actually will need to uncomment this to make current workflow uninterrupted, thanks for catching